scripts/scb_wrapper.py

```diff
@@ -66,6 +67,15 @@ def __init__(self, cert_path=CERT_PATH) -> None:
         self.json = {}
         self.update_owned_vars()
         self.reset_json()
+        
+    def init_session(self):
```

Comment:
Should be a singelton:
```suggestion
    def get_session(self):
         if self.session is None:
             self.session = ... new session ...
         return self.session
```